lib/src/css_mirroring/settings.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+part of scissors.src.css_mirroring.transformer;
+
+enum Direction { ltr, rtl }
+
+Direction flipDirection(Direction dir) =>
+    dir == Direction.ltr ? Direction.rtl : Direction.ltr;
+
+abstract class CssMirroringSettings {
+  Setting<bool> get verbose;
+
+  final mirrorCss =
+      new Setting<bool>('mirrorCss', debugDefault: true, releaseDefault: true);
```

Comment:
as discussed, `defaultValue: false` (until we can disable it only for the scissors transformer and not the mirroring one)